src/bridge/fetch.ts

```diff
@@ -8,6 +8,8 @@ import { URL, parse as parseURL, format as formatURL } from 'url'
 import { fullURL } from '../utils/http'
 import { transferInto } from '../utils/buffer'
 import { ProxyStream } from './proxy_stream'
+import * as zlib from 'zlib'
+import { Readable } from 'stream'
```

Comment:
Does not seem like this is used anymore.

Reply:
Fixed. Got rid of Readable (zlib is still used).